package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "azure-pipelines-tool-lib",
-  "version": "0.13.2",
+  "version": "1.0.0",
```

Comment:
I think this needs to be `1.0.1` to be higher than the current published `1.0.0-preview.0`

Reply:
Thanks! Fixed it